### PR TITLE
Add 24 as a supported release branch

### PR DESCRIPTION
**Pull request by mitalipaygude**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by vignesh-goutham — APPROVED** [review]
/approve

File: projects/aws/image-builder/builder/builder.go [hunk 1]
```diff
@@ -141,7 +140,8 @@ func (b *BuildOptions) ValidateInputs() {
 		log.Fatalf("Error converting vsphere config path to absolute")
 	}
 
-	if b.ReleaseChannel != "1-20" && b.ReleaseChannel != "1-21" && b.ReleaseChannel != "1-22" && b.ReleaseChannel != "1-23" {
-		log.Fatalf("release-channel should be one of 1-20, 1-21, 1-22, 1-23")
+	supportedReleaseBranches := GetSupportedReleaseBranches()
+	if !SliceContains(supportedReleaseBranches, b.ReleaseChannel) {
+		log.Fatalf("release-channel should be one of 1-20, 1-21, 1-22, 1-23, 1-24")
```
Comment (abhay-krishna) [inline projects/aws/image-builder/builder/builder.go:145]:
```suggestion
		log.Fatalf("release-channel should be one of %v", supportedReleaseBranches)
```

File: projects/aws/image-builder/builder/utils.go [hunk 2]
```diff
@@ -40,3 +42,55 @@ func cleanup(buildToolingDir string) {
...
+	}
+	return buildToolingRepoPath
+}
+
+func getRepoRoot() (string, error) {
+	cmd := exec.Command("git", "rev-parse", "--show-toplevel")
+	return ExecCommand(cmd)
+}
+
+func ExecCommand(cmd *exec.Cmd) (string, error) {
```
Comment (abhay-krishna) [inline projects/aws/image-builder/builder/utils.go:72]: Let's use `execCommandWithStreamOutput` that's already declared above.

**Pull request merged**